docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -60,6 +60,10 @@ Note: On a system with SELinux enabled, the generated files will inherit context
 
 Print the created units in specified format (json). If `--files` is specified the paths to the created files will be printed instead of the unit content.
 
+#### **--header**=*header*
+
+Append the specified header. This option can be used with `--no-header` to generate the specified header only.
+
```

Comment:
I'm a bit lost as to what the header is and where it's being appended to.  I think it's a "systems section header" and it's being appended to the systems drop-in file, but I'm not sure.  Can you clarify?  Also, I'm unclear on what action is taken when both --header and --no-header are specified.  Further clarification on that would be helpful too.

Reply:
Please check these examples. It would be great if anyone could complete this PR.
https://github.com/containers/podman/pull/15291#issuecomment-1248318217